examples/image_segmenter/README.md

```diff
@@ -18,90 +18,18 @@ wget https://download.pytorch.org/models/fcn_resnet101_coco-7ecb50ca.pth
-   10.846567  ]
-  [11.49452    11.49452    11.49452    ... 10.846567   10.846567
-   10.846567  ]
-  ...
-  [10.028987   10.028987   10.028987   ...  9.980104    9.980104
-    9.980104  ]
-  [10.028987   10.028987   10.028987   ...  9.980104    9.980105
-    9.980105  ]
-  [10.028987   10.028987   10.028987   ...  9.980104    9.980105
-    9.980105  ]]
-
- [[-2.524181   -2.524181   -2.524181   ... -1.4157648  -1.4157648
-   -1.4157648 ]
-  [-2.524181   -2.524181   -2.524181   ... -1.4157648  -1.4157648
-   -1.4157648 ]
-  [-2.524181   -2.524181   -2.524181   ... -1.4157648  -1.4157648
-   -1.4157648 ]
-  ...
-  [-0.52271044 -0.52271044 -0.52271044 ... -0.910931   -0.910931
-   -0.910931  ]
-  [-0.52271044 -0.52271044 -0.52271044 ... -0.910931   -0.910931
-   -0.910931  ]
-  [-0.52271044 -0.52271044 -0.52271044 ... -0.910931   -0.910931
-   -0.910931  ]]
-
- [[-1.1300591  -1.1300591  -1.1300591  ... -0.88538504 -0.88538504
-   -0.88538504]
-  [-1.1300591  -1.1300591  -1.1300591  ... -0.88538504 -0.88538504
-   -0.88538504]
-  [-1.1300591  -1.1300591  -1.1300591  ... -0.88538504 -0.88538504
-   -0.88538504]
-  ...
-  [-1.1726367  -1.1726367  -1.1726367  ... -1.6144376  -1.6144376
-   -1.6144376 ]
-  [-1.1726367  -1.1726367  -1.1726367  ... -1.6144376  -1.6144376
-   -1.6144376 ]
-  [-1.1726367  -1.1726367  -1.1726367  ... -1.6144376  -1.6144376
-   -1.6144376 ]]
-
- ...
-
- [[-0.27236405 -0.27236405 -0.27236405 ... -0.791381   -0.79138106
-   -0.79138106]
-  [-0.27236405 -0.27236405 -0.27236405 ... -0.791381   -0.79138106
-   -0.79138106]
-  [-0.27236405 -0.27236405 -0.27236405 ... -0.791381   -0.79138106
-   -0.79138106]
-  ...
-  [-0.16645516 -0.16645516 -0.16645516 ...  0.4377911   0.4377911
-    0.4377911 ]
-  [-0.16645516 -0.16645516 -0.16645516 ...  0.4377911   0.4377911
-    0.4377911 ]
-  [-0.16645516 -0.16645516 -0.16645516 ...  0.4377911   0.4377911
-    0.4377911 ]]
-
- [[-0.70757735 -0.70757735 -0.70757735 ... -1.0088179  -1.0088179
-   -1.0088179 ]
-  [-0.70757735 -0.70757735 -0.70757735 ... -1.0088179  -1.0088179
-   -1.0088179 ]
-  [-0.70757735 -0.70757735 -0.70757735 ... -1.0088179  -1.0088179
-   -1.0088179 ]
-  ...
-  [ 0.44139242  0.44139242  0.44139242 ...  0.28535858  0.28535858
-    0.28535858]
-  [ 0.44139242  0.44139242  0.44139242 ...  0.28535858  0.28535858
-    0.28535858]
-  [ 0.44139242  0.44139242  0.44139242 ...  0.28535858  0.28535858
-    0.28535858]]
-
- [[-0.47415262 -0.47415262 -0.47415262 ... -0.4314881  -0.43148813
-   -0.43148813]
-  [-0.47415262 -0.47415262 -0.47415262 ... -0.4314881  -0.43148813
-   -0.43148813]
-  [-0.47415262 -0.47415262 -0.47415262 ... -0.4314881  -0.43148813
-   -0.43148813]
-  ...
-  [ 0.04289126  0.04289126  0.04289126 ... -0.42638034 -0.42638034
-   -0.42638034]
-  [ 0.04289126  0.04289126  0.04289126 ... -0.42638034 -0.42638034
-   -0.42638034]
-  [ 0.04289126  0.04289126  0.04289126 ... -0.42638034 -0.42638034]
-```
+[[
```

Comment:
Will the user be able to repro segmented image with this response?

Reply:
Updated this doc with the same output from the example.